presto-geospatial/src/main/java/com/facebook/presto/plugin/geospatial/GeoFunctions.java

```diff
@@ -356,6 +387,26 @@ public static Slice stStartPoint(@SqlType(GEOMETRY_TYPE_NAME) Slice input)
         return serialize(createFromEsriGeometry(lines.getPoint(0), reference));
     }
 
+    @Description("Returns a \"simplified\" version of the given geometry")
+    @ScalarFunction("Simplify_geometry")
+    @SqlType(GEOMETRY_TYPE_NAME)
+    public static Slice simplifygeometry(@SqlType(GEOMETRY_TYPE_NAME) Slice input,
+                                         @SqlType(DOUBLE) double distanceTolerance) {
+        if (Double.isNaN(distanceTolerance)) {
+            throw new PrestoException(INVALID_FUNCTION_ARGUMENT, "distance is NaN");
+        }
+
+        if (distanceTolerance < 0) {
+            throw new PrestoException(INVALID_FUNCTION_ARGUMENT, "distance is negative");
+        }
+
+        if (distanceTolerance == 0) {
+            return input;
+        }
+
+        return JtsGeometryUtils.serialize(JtsGeometryUtils.simplify(input, distanceTolerance));
```

Comment:
please, inline the call to `simplify`

```
JtsGeometryUtils.serialize(TopologyPreservingSimplifier.simplify(JtsGeometryUtils.deserialize(input), distanceTolerance));
```

consider, static import of TopologyPreservingSimplifier.simplify